unlock-app/src/hooks/useCheckoutCommunication.ts

```diff
@@ -155,8 +157,11 @@ export const useCheckoutCommunication = () => {
   const emitOnEvent = (eventName: string) => {
     pushOrEmit(CheckoutEvents.onEvent, eventName)
   }
+
   // If the page is not inside an iframe, window and window.top will be identical
-  const insideIframe = window.top !== window
+  if (typeof window !== 'undefined') {
+    insideIframe = window.top !== window
+  }
```

Comment:
This is a change to accommodate SSR. This won't result in any change in the functionality.